ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-detail.html

```diff
@@ -138,7 +138,7 @@
                     <md-divider></md-divider>
                     <md-list-item>
                         <div layout="column"  class="item-column md-list-item-text">
-                            <span class="item-title">{{vm.service.checkDate | date : "MM/dd/yyyy 'at' h:mm"}}</span>
+                            <span class="item-title">{{'service-component-detail.time-locale' | translate:{date:vm.service.checkDate} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:vm.service.checkDate} }}</span>
```

Comment:
There are no English translations for `service-component-detail.at` and `service-component-detail.time-locale2`. Please fix this to only use `service-component-detail.time-locale`.